shared/js/ui/templates/shared/hero.es6.js

```diff
@@ -3,22 +3,26 @@ const bel = require('bel')
 module.exports = function (ops) {
     const slidingSubviewClass = ops.showClose ? 'js-sliding-subview-close' : ''
     return bel`<div class="hero border--bottom text--center ${slidingSubviewClass} silver-bg">
-    ${renderOpenOrCloseButton(ops.showClose)}
     <div class="hero__icon hero__icon--${ops.status}">
     </div>
     <h1 class="hero__title">
         ${ops.title}
     </h1>
-    <h2 class="hero__subtitle">
+    <h2 class="hero__subtitle" aria-label="${ops.subtitleLabel ? ops.subtitleLabel : ops.subtitle}">
         ${ops.subtitle}
     </h2>
+    ${renderOpenOrCloseButton(ops.showClose)}
```

Comment:
Moved this at the bottom, because it doesn't make sense to hear "more details" before you've heard what "more details" actually refers to.